templates/ui.html

```diff
@@ -2131,9 +2158,45 @@
           }
         },
         mounted() {
-          this.updateDefaultOptions("default", this.initializeFilerobot);
+          this.updateDefaultOptions(
+              "default",
+              () => {this.initializeFilerobot(), this.startIntroJS()}
```

Comment:
Should it be `;` in `this.initializeFilerobot(); this.startIntroJS();`, instead of `,`?